tests/sys/vane/ford.hoon

```diff
@@ -7580,4 +7581,33 @@
       [%leaf "builds.state:"]
       [%rose braces build-state]
   ==
+:: +expect-ford-very-empty: assert that ford's state is a very empty ship
+::
+::   At the end of some tests, we want to assert that we have cleaned up all
+::   state, *including* the caches.
+++  expect-ford-very-empty
+  |=  [ford-gate=_ford-gate ship=@p]
+  ^-  tang
+  ::
+  =/  ford  *ford-gate
+  =/  state  state.ax.+>+<.ford
+  =/  default-state  *ford-state:ford
+  ::
+  ?:  =(default-state state)
```

Comment:
Instead of duplicating this code in `+expect-ford-very-empty` and `+expect-ford-empty`, I think the latter should call the former.